source/conf.py

```diff
@@ -36,6 +36,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = ['sphinx.ext.autodoc']
+extensions = ['sphinx.ext.autosectionlabel']
```

Comment:
Shouldn't this be 
```python
extensions = ['sphinx.ext.autodoc', 'sphinx.ext.autosectionlabel']
```
It seems to me that we are overriding the previous line otherwise
@fabiencastan